src/Storages/ObjectStorage/StorageObjectStorageSettings.h

```diff
@@ -44,33 +44,6 @@ class SettingsChanges;
     M(CLASS_NAME, UInt64Auto) \
     M(CLASS_NAME, URI)
 
-// clang-format off
-
-#define STORAGE_OBJECT_STORAGE_RELATED_SETTINGS(DECLARE, ALIAS) \
```

Comment:
Stop, did we just remove all object storage settings? Do we need this class at all now?